python/plugins/processing/tests/testdata/expected/grid_diamond.gml

```diff
@@ -1,214 +0,0 @@
-<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
It makes me nervous seeing the existing tests removed. Is there any reason these could not be retained? If not, have you checked to see what the differences in the resultant layers are?